blog/content/post/2023-01-23-3mdeb-insights-winter-2023.md

```diff
@@ -0,0 +1,122 @@
+---
+title: 3mdeb Insights - Winter 2023
+abstract: 'This blog will provide an in-depth look at upcoming plans and events
+          for the near future.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2023-01-23
+archives: "2023"
+
+tags:
+  - Insights
+  - Events
+  - Open-source contribution
```

Comment:
@Asiderr rather than creating new tags, please use previously added: https://blog.3mdeb.com/tags/, for example `opensource`

Reply:
I changed tags, and I add a new one. 083a71e